tests/integration/integration_test_Entity.py

```diff
@@ -141,7 +141,7 @@ def test_Entity():
 
     newfolder = Folder('Testing Folder', parent=project)
     newfolder = syn.store(newfolder)
-    link = Link(newfolder.id, parent = folder.id)
```

Comment:
So this shouldn't matter anymore, as I fixed the Link entity to accept both Synapse ids or entities.